test/unit/model/virtualbox-test.js

```diff
@@ -240,10 +241,17 @@ describe('Virtualbox installer', function() {
       }
     });
 
+    it('should able to check file were extracted to temp', function(done) {
+      sandbox.stub(child_process, 'execFile').yields(null, "Files were extracted to temp", "");
+      item2.setInstallComplete();
+      item2.thenInstall(installer);
+      installer.install(fakeProgress, success, failure);
+      done();
```

Comment:
if you use done callback from mocha, you should be calling it from callback passed to installer.installAfterRequirements().

``` JavaScript
installer.installAfterRequirements(fakeProgress, function (){
 expect().something.has.been.done;
 done();
 }, function(){
  expect.fail();
 });
```
that is needed only if you really expect async code is still going to be running when test method exits.

if you mock all async methods then you don't need use done callback at all, just `expect().test.the.condition;` should do the trick.